.github/workflows/release.yml

```diff
@@ -1,10 +1,10 @@
 # mostly copied from https://raw.githubusercontent.com/web-infra-dev/oxc/main/.github/workflows/release_cli.yml
 name: release
 on:
-  push:
-    branches: ["main"]
-  release:
-    types: [created]
+  workflow_run:
+    workflows: [test]
```

Comment:
This will require us to fix #66 before the next release, right?

Reply:
Yeah, think so. I tried to solve this on my local windows machine, and got it to test and build with this diff: https://github.com/keithamus/hdx/pull/74 which I can see is failing now with some CRLF things maybe which is strange, because did not locally.